src/ui/styles/index.scss

```diff
@@ -14,22 +14,24 @@
 
 /* All molecule ui components should live here */
 @import './BurgerMenu';
-@import './CoverLetter.scss';
-@import'./ExpandingTextField.scss';
+@import './CoverLetter';
+@import'./ExpandingTextField';
 @import './ExcludedToggle';
 @import './ExpandingEmailField';
 @import './FileUpload';
 @import './Modal';
 @import './Menu';
-@import './MultiFileUpload.scss';
+@import './MultiFileUpload';
 @import './PersonInfo';
 @import './PersonPod';
 @import './ProfileDropdown';
 @import './ProgressBar';
 @import './SelectedOption';
 @import './SelectedPeopleList';
-@import './Toggle.scss';
-@import './UploadProgress.scss';
+@import './Toggle';
+@import './UploadProgress';
 
 /* All organism ui components should live here */
 @import './PeoplePicker';
+
+@import './RichTextEditor';
```

Comment:
should be moved into the molecules section

Reply:
done